examples/custom-admin-ui-pages/admin/pages/custom-page.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsx jsx */
+import { Fragment } from 'react';
+import { jsx } from '@keystone-ui/core';
+// Please note that while this capability is driven by Next.js's pages directory
```

Comment:
Would be good to show how to import global css that needs to be used by third party library...

I just tried to bring in [`antd`](https://ant.design/docs/react/getting-started#2.-Use-and-modify-an-antd-component) and you get an error:

> Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules

Reply:
[this ](https://github.com/keystonejs/keystone/blob/5d6197f671a46a913d843e36ad50a92eb4f8bf9f/examples-next/basic/admin/config.tsx#L17) worked for me then, not sure if that is a problem now.

Reply:
@gautamsi interesting... 

Kinda a work around as I'd have to copy the styles to `public` or find a CDN for the library at hand.

Would be better if there was some way to hook into the `_app.js`.